measure.rb

```diff
@@ -2805,21 +2806,33 @@ def self.add_ceiling_fans(runner, model, building, weather)
                             @cfa, @living_space)
   end
 
-  def self.check_distribution_system(building, system_values)
-    dist_id = system_values[:distribution_system_idref]
+  def self.check_distribution_system(building, dist_id, system_id, system_type)
     return if dist_id.nil?
 
+    hvac_distribution_type_map = { "Furnace" => ["AirDistribution", "DSE"],
+                                   "Boiler" => ["HydronicDistribution", "DSE"],
+                                   "central air conditioner" => ["AirDistribution", "DSE"],
+                                   "air-to-air" => ["AirDistribution", "DSE"],
+                                   "mini-split" => ["AirDistribution", "DSE"],
+                                   "ground-to-air" => ["AirDistribution", "DSE"] }
```

Comment:
Hmm, why is evaporative cooler not in this list? It is similar to mini-split in that it can be optionally ducted.

Reply:
Yeah the nil check is also for skipping evap cooler. Evap cooler didn't check system type in EP-validator so I didn't address it here. I will add it to both places.